packages/nocodb-sdk/src/lib/sqlUi/MysqlUi.ts

```diff
@@ -1015,6 +1011,11 @@ export class MysqlUi {
         break;
       case 'SingleLineText':
         colProp.dt = 'varchar';
+        colProp.validate = {
+          func: ['isLength'],
+          args: [''],
+          msg: ['Validation failed : isLength ({cn})'],
+        };
```

Comment:
Is it required? We are not passing the max value argument at all here 

Ref: https://www.npmjs.com/package/validator

Reply:
Oh, I thought it would infer max length if not passed. Will remove it